### DISCUSSION TITLE
What template to use to co-register lesion in MNI space

### DISCUSSION THREAD

**Discussion by frankatimm** [simnibs/simnibs#330] [7.Creating head models]
Hello everyone, 

I am currently trying to add individual lesions to Ernie but the lesion (MNI space) doesn't have the right orientation after I run the meshing and segmentation (it is kind of shifted to the back). Now I am wondering: is Ernie in MNI space? if no, is there a template available in MNI space? Is it the MNI_152 folder? How can I proceed from there? Thank you in advance for your help.

Best,
Franka

**Comment by bjornsig** [comment]
Hello Franks,
No, ernie is not in MNI space but you can transform your mask from MNI space to ernie with:
https://simnibs.github.io/simnibs/build/html/documentation/command_line/mni2subject.html#mni2subject

make sure to use --interpolation_order 0
